### PR TITLE
Use current guidance for NOIRLab observatory affiliations

### PR DESCRIPTION
**Pull request by timj**

### CODE REVIEW

**Comment by timj** [comment]
@ranpalgill / @womullan my understanding from the library page and the newsletter is that Bob and Cathy are now RubinOps and doesn't need a separate NOIRLab affiliation. @ranpalgill I think if you are construction and ops equally then it likely should be RubinOpsC and RubinObsC?

File: etc/authordb.yaml [hunk 1]
```diff
@@ -166,7 +165,8 @@ affiliations:
   NCNRP: National Centre of Nuclear Research, Andrzeja So\l{}tana 7, 05-400 Otwock, Poland
   NCSA: NCSA, University of Illinois at Urbana-Champaign, 1205 W.\ Clark St., Urbana,
     IL 61801, USA
-  NOIRLab: NSF's National Optical-Infrared Astronomy Research Laboratory, 950 N.\ Cherry Ave., Tucson, AZ 85719, USA
+  NOIRLab: NSF NOIRLab, 950 N.\ Cherry Ave., Tucson, AZ 85719, USA
+  NOIRLabC: NSF NOIRLab, Casilla 603, La Serena, Chile
```
Comment (timj) [inline etc/authordb.yaml:169]: "Casilla 603" does seem to be the contact address listed on the website. Google maps can't find it though so it seems like an odd choice but that is what we are told to use.

**Comment by ranpalgill** [comment]
> @ranpalgill / @womullan my understanding from the library page and the newsletter is that Bob and Cathy are now RubinOps and doesn't need a separate NOIRLab affiliation. @ranpalgill I think if you are construction and ops equally then it likely should be RubinOpsC and RubinObsC?

I am officially RubinOpsC

**Review by womullan — APPROVED** [review]
thanks looks good

**Pull request merged**